pkg/virt-operator/resource/generate/components/webhooks.go

```diff
@@ -355,6 +354,12 @@ func NewVirtAPIValidatingWebhookConfiguration(installNamespace string) *admissio
 						Path:      &launcherEvictionValidatePath,
 					},
 				},
+				MatchConditions: []admissionregistrationv1.MatchCondition{
+					{
+						Name:       "only-vms",
+						Expression: `object.metadata.name.startsWith("virt-launcher")`,
```

Comment:
Why don't we rely on labels with something like:
```suggestion
						Expression: `object.metadata.labels.filter(l, l.name = 'kubevirt.io').all(l, l.value == 'virt-launcher')`,
```

Reply:
I prefer to keep it as is for now. For the future this should be straightforward change.